pkg/store/storepb/rpc.proto

```diff
@@ -39,6 +40,21 @@ service Store {
   rpc LabelValues(LabelValuesRequest) returns (LabelValuesResponse);
 }
 
+/// WriteableStore reprents API against instance that stores XOR encoded values with label set metadata (e.g Prometheus metrics).
+service WriteableStore {
+  // WriteRequest allows you to write metrics to this store via remote write
+  rpc RemoteWrite(WriteRequest) returns (WriteResponse) {}
+}
+
+message WriteResponse {
+}
+
+message WriteRequest {
+  string tenant = 1;
```

Comment:
tenant and replica are currently not used at all by the RemoteWrite rpc, as it doesn't check and forward further. Are they placeholders or is the plan to use them?

Reply:
Yep they’ll be used to forward requests. I gotta continue that part tomorrow 🙂